addOns/ascanrules/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
 ### Changed
+- Cloud Metadata Scan Rule now supports multiple cloud providers. Implemented provider-specific metadata indicators to reduce false positives.
```

Comment:
It already supported multiple providers, better be specific. The latter part should be under Fixed.

Reply:
Also, mention the change in confidence per the latter fix.